docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -185,8 +185,12 @@ public class MyWebSocket {
 
 ==== Request context
 
-If an endpoint is annotated with `@RequestScoped`, or with a security annotation (such as `@RolesAllowed`), or depends directly or indirectly on a `@RequestScoped` bean, or on a bean annotated with a security annotation, then each WebSocket endpoint callback method execution is associated with a new _request context_.
-The request context is active during endpoint callback invocation.
+Each WebSocket endpoint callback method execution is associated with a new _request context_ (activate CDI request context) if the endpoint is:
```

Comment:
```suggestion
Each WebSocket endpoint callback method execution is associated with a new CDI _request context_ if the endpoint is:
```

I'm just not sure how to change `activate CDI request context` since `activate` is a verb